app/components/DecredInput/DcrInput.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Balance from "../Balance";
+import NumericInput from "../NumericInput";
+
+const DcrInput = ({
+  submited,
+  invalidMessage,
+  requiredMessage,
+  required,
+  invalid,
+  value,
+  placeholder,
+  onChange,
+  disabled,
+  readOnly,
+  getCurrencyValue
+}) => {
+  return (
+    <div>
+      <NumericInput
+        className="stakepool-content-nest-purchase-ticket-form"
+        submited= {submited}
+        invalidMessage= {invalidMessage}
+        requiredMessage= {requiredMessage}
+        disabled={disabled}
+        readOnly={readOnly}
+        placeholder={placeholder}
+        required= {required}
+        invalid= {invalid}
+        value={value}
+        onChange={onChange}
+      />
+      <Balance
```

Comment:
I don’t think we need to show the balance again but we should show the unit type and eventually a drop down for atoms vs DCR or other future units.

Reply:
What drop down would be that?